docs/_guide/styles.md

```diff
@@ -113,19 +113,19 @@ To define a static `styles` property:
 
 Static styles apply to all instances of an element. Any expressions in your CSS are evaluated and included **once**, then reused for all instances. 
 
-For security reasons, expressions must be tagged with the `cssLiteral` template literal tag:
+For security reasons, use unsafeCSS to pass non-literal values using only trusted sources:
 
 ```js
-import { LitElement, css, cssLiteral } from 'lit-element';
+import { unsafeCSS } from 'lit-element';
```

Comment:
`LitElement` is used in the example so it should still be imported.

Reply:
Fixed the import to include LitElement in latest commit.